pkgs/development/libraries/fastpbkdf2/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, openssl }:
+
+stdenv.mkDerivation {
+  name = "fastpbkdf2-1.0.0";
+  
+  src = fetchurl {
+    url = https://github.com/ctz/fastpbkdf2/archive/v1.0.0.tar.gz;
```

Comment:
This seems to be ref archive, these are not deterministic (because compression is not deterministic). Hash will change after a while. Use `fetchFromGitHub` instead.